src/SweetAlert/SweetAlertNotifier.php

```diff
@@ -223,4 +223,20 @@ private function hasTitle()
     {
         return (bool) strlen($this->config['title']);
     }
+    
+    /**
+     * Make Message HTML view.
+     * 
+     * @param bool|true $html
+     *
+     * @return \UxWeb\SweetAlert\SweetAlertNotifier $this
+     */
+    public function html($html = true)
```

Comment:
Thank you @uxweb sorry it was mistake i have added while testing on my local machine and forget to remove it. i have commit update :+1: